esp/templates/program/modules/teacheronsite/schedule.html

```diff
@@ -0,0 +1,45 @@
+{% extends "program/modules/teacheronsite/webapp.html" %}
+
+{% block body %}
+
+<center>
+
+  <div class="main">
+    <table class="teacherschedule">
+      <thead>
+        <th colspan="3" class="user_info">
+            <i>Classes for {{user.first_name}} {{user.last_name}} - ID: {{user.id}}</i>
+            {% if not checked_in %}
+                <p class="checkinnote">Note: Please make sure to check in before your first class today.</p>
+            {% endif %}
```

Comment:
I suggest making this text customizable with a tag, so that you don't have to do a template override just to change this message.